scripts/reset-password.py

```
@@ -0,0 +1,233 @@
+"""Reset a user's password via CLI.
```

Ruff (0.15.14)

[warning] The file is executable but no shebang is present

(EXE002)